src/site/views/event/tmpl/default.php

```diff
@@ -101,7 +102,73 @@
 							</tr>
 							</tfoot>
 						</table>
+						<table class="table table-hover">
+							<thead>
+							<tr>
+								<th>University</th>
+								<th>Tickets Sold</th>
+							</tr>
+							</thead>
+							<tbody>
+							<?php
+							$universityCounts = array();
+
+							foreach ($eventAttendees as $person)
+							{
+								$universityCounts[$person['Uni']] += 1;
+							}
+							arsort($universityCounts);
+							foreach ($universityCounts as $uni => $count)
+							{
+								echo "<tr>\n";
+								echo "<td>{$uni}</td>";
+								echo "<td>{$count}</td>";
+								echo "</tr>\n";
+							}
+							?>
+							</tbody>
+						</table>
+
+						<?php
+						  // Only want to show the full attendees table when the club committees actually need it
```

Comment:
And when is it that they actually need it? Be good to say this explicitly.

Reply:
the date of event ticket sales opening isn't exposed here, so I'll just leave this if statement open-ended at the start, and if there are no tickets it'll just revert to "There is no ticket information to show at this time."

Reply:
True, would need to add `event.event_date_open` to `getItem()` or join on the `event` table in `getEventAttendees()`. Having "There is no ticket information to show at this time." works too. 👍